doc/howtos/rdtraining/setup.rst

```diff
@@ -0,0 +1,408 @@
+.. _howto/rdtraining/setup:
+
+==============================
+Development Environment Set-up
+==============================
+
+There are multiple ways to install Odoo depending on the intended use case.
+
+This documents attempts to describe the installation option for an internal Odoo R&D developer. We
+assume that you are installing your development environment in an Odoo's standard laptop with Linux
+Mint installed and updated. If you another environment is used, you can refer to :ref:`setup/install/source`
+
+Fetch the sources & configure Git
+=================================
+
+Install and configure git
+-------------------------
+
+.. code-block:: console
+
+    $ sudo apt install git
+    $ git config --global user.name "Your full name"
+    $ git config --global user.email "xyz@odoo.com"
+
+Configure github
+----------------
+
+To fetch the sources and to contribute in the Odoo's development you will need a GitHub user. If you
+don't have already a GitHub user, we will recommend using your trigram (xyz) followed by '-odoo':
+'xyz-odoo'.
```

Comment:
I would do the other way around: suggest to use a specific user for Odoo, but if prefer they use their own.